src/components/layout/layout.tsx

```diff
@@ -99,7 +101,7 @@ function overrideDarkFromQuery() {
   return params.has('dark');
 }
 
-const Layout: React.FC<LayoutProps> = (props) => {
+const Layout: FunctionComponent<LayoutProps> = (props) => {
```

Comment:
Recommendation by react team: Use `JSX.Element`.

```
const Layout = (props: PropsType): JSX.Element => (...)
```

Reply:
I think this is rule number one with react & TS: You will always pick the wrong type if you don't look it up once to learn the correct one 😬Thanks.

Reply:
Can you link to the recommendation?
`React.FC` and `React.FunctionComponent` are the same types and they refer to functional components and include the children property. When I switch to the `JSX.Element` type I have to declare the children manually which is awkward. It's working so we can keep it and try it out in the next PR.

Reply:
Declarin children manually is intentional. Not all components have children.

- There is a more detailed blog post about that here: https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/
- And the React PR that even removes the `React.FC` from types: https://github.com/facebook/create-react-app/pull/8177 (this is from 2020 and it's still in the code, so I don't know where exactly they removed it... 😄 )

Reply:
Thanks for the links, I'm finally convinced and I will be explicit about everything including children in the future 🤓🙌